rust-toolchain

```diff
@@ -1 +1 @@
-nightly-2020-05-07
\ No newline at end of file
+stable
```

Comment:
Should we remove this file, or is there some benefit to locking it to stable?

Reply:
yes this no longer needed

Reply:
commit 0cc483b removes;